### PR TITLE
Data Model

### PR DESCRIPTION
**Pull request by Retky**
image

### 📝 [Requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_1.md):

- ✔️ Create a new app. Give it a meaningful name.
- ✔️ Initialize your project with Git.
- ✔️ Make sure that your project has a Postgres database set up.
- ✔️ Build your project schema.
    - ✔️ Create and run the necessary migration files.
    - ✔️ Table and column names should match the ERD diagram.
        - ✔️ note: photo for users table should be a link to an image
    - ✔️ Foreign keys should be included.
    - ✔️ All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20220705212239_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :author_id, null: false
```
Comment (mikethreels) [inline db/migrate/20220705212239_create_comments.rb]: - [x] this part of the migration should look something like the below:
```
t.references :author, foreign_key: { to_table: 'users' }
```

File: db/migrate/20220705211136_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :author_id, null: false
```
Comment (mikethreels) [inline db/migrate/20220705211136_create_posts.rb]: - [x] this part of the migration should look something like the below:
```
t.references :author, foreign_key: { to_table: 'users' }
```

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Retky, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220705212239_create_comments.rb (hunk 1 above)
Comment (Retky) [reply db/migrate/20220705212239_create_comments.rb]: Thank you for the feedback, ill implement it

File: db/migrate/20220705211136_create_posts.rb (hunk 2 above)
Comment (Retky) [reply db/migrate/20220705211136_create_posts.rb]: Thank you for the feedback, ill implement it

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Retky  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**